config/ConnectorConfig.json

```diff
@@ -9,7 +9,8 @@
    },
    "AppConfig":{
       "CycleTimeMs":1000,
-      "PostDuplicateTagValues": false
+       "PostDuplicateTagValues": false,
+       "OMF":false
    },
    "TagList":["ExampleTag1", "ExampleTag2", "ExampleTag3", "ExampleTag4"]
-}
\ No newline at end of file
+}

```

Comment:
this would be a separate commit to add the newline at the end of the file since its a whitespace only change

Reply:
I dont know why but i cant take this out, so i gave it it's own commit

Reply:
its just because you have a newline at the end of the file line 16.